### PR TITLE
GUVNOR-3062: Add tests for selection models

### PR DESCRIPTION
**Pull request by jomarko**
Tests for #485

### CODE REVIEW

File: drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/test/java/org/drools/workbench/screens/guided/dtable/client/wizard/table/pages/ActionInsertFactFieldsPageViewImplTest.java [hunk 1]
```diff
@@ -0,0 +1,95 @@
+package org.drools.workbench.screens.guided.dtable.client.wizard.table.pages;
```
Comment (manstis) [inline drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/test/java/org/drools/workbench/screens/guided/dtable/client/wizard/table/pages/ActionInsertFactFieldsPageViewImplTest.java]: License please

File: drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/test/java/org/drools/workbench/screens/guided/dtable/client/wizard/table/pages/ActionSetFieldsPageViewImplTest.java [hunk 2]
```diff
@@ -0,0 +1,71 @@
+package org.drools.workbench.screens.guided.dtable.client.wizard.table.pages;
```
Comment (manstis) [inline drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/test/java/org/drools/workbench/screens/guided/dtable/client/wizard/table/pages/ActionSetFieldsPageViewImplTest.java]: License please

File: drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/test/java/org/drools/workbench/screens/guided/dtable/client/wizard/table/pages/FactPatternConstraintsPageViewImplTest.java [hunk 3]
```diff
@@ -0,0 +1,66 @@
+package org.drools.workbench.screens.guided.dtable.client.wizard.table.pages;
```
Comment (manstis) [inline drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/test/java/org/drools/workbench/screens/guided/dtable/client/wizard/table/pages/FactPatternConstraintsPageViewImplTest.java]: License please

File: drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/test/java/org/drools/workbench/screens/guided/dtable/client/wizard/table/pages/FactPatternsPageViewImplTest.java [hunk 4]
```diff
@@ -0,0 +1,60 @@
+package org.drools.workbench.screens.guided.dtable.client.wizard.table.pages;
```
Comment (manstis) [inline drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/test/java/org/drools/workbench/screens/guided/dtable/client/wizard/table/pages/FactPatternsPageViewImplTest.java]: License please

**Comment by jomarko** [comment]
Licences added.

**Comment by manstis** [comment]
Jenkins please retest this.

**Pull request merged**